### ISSUE TITLE
Install docs path

### ISSUE THREAD

**Issue by mstormi** [openhab-scripters/openhab-helper-libraries#285]
Docs say to install jython jar into what's in EXTRA_JAVA_OPTS, i.e. /etc/openhab2/automation/jython/, but that dir does not exist. Either a user must create it or this is a docs error.

**Comment by openhab-5iver** [comment]
Good point... I'll mention that they need to create it. Glad to see you dipping your toe in!
